Firestore/Example/Tests/Remote/FSTSerializerBetaTests.m

```diff
@@ -396,20 +396,25 @@ - (void)testRoundTripSpecialFieldNames {
 
 - (void)testEncodesListenRequestLabels {
   FSTQuery *query = FSTTestQuery(@"collection/key");
-  FSTQueryData *queryData =
-      [[FSTQueryData alloc] initWithQuery:query targetID:2 purpose:FSTQueryPurposeListen];
+  FSTQueryData *queryData = [[FSTQueryData alloc] initWithQuery:query
+                                                       targetID:2
+                                           listenSequenceNumber:2
```

Comment:
Making this a different value than the targetID would help catch cases where we propagate the wrong property via the serializer.

Reply:
Done.